### ISSUE TITLE
Bug in Latest version 1.0.11

### ISSUE THREAD

**Issue by arvinep** [patrick96/click-link#10]
After update the packge from 1.0.10 to latest one in Atom, I got this error:

```
TypeError: Cannot read property 'paneContainer' of undefined
    at Object.activate (/home/arvin/.atom/packages/click-link/lib/click-link.js:11:43)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:218:19)
    at /usr/share/atom/resources/app.asar/src/package.js:190:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:96:15)
    at /usr/share/atom/resources/app.asar/src/package.js:183:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:180:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:550:34)
    at /usr/share/atom/resources/app.asar/src/package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (/usr/share/atom/resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/usr/share/atom/resources/app.asar/src/package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (/usr/share/atom/resources/app.asar/src/package-manager.js:508:46)
    at /usr/share/atom/resources/app.asar/src/atom-environment.js:826:28
```

**Workaround:**
Temporary I replaced following line in `/home/arvin/.atom/packages/click-link/lib/click-link.js` file:

From:
`atom.workspace.activePaneContainer.paneContainer.element.addEventListener("click", function(e) {`

To:
`document.querySelector("atom-pane-container.panes").addEventListener("click", function(e) {`

Would be great to push the fix.

**Comment by patrick96** [comment]
Aw man, until a few weeks ago I had that exact code until I changed it in d1dbeb92e739d95fd21356948399dd728e0bc762 because the eventListener wasn't working anymore.

I'll look into it in a few hours and try to reproduce it.

**Comment by patrick96** [comment]
Alright I wasn't able to reproduce this error. Which version of atom are you using?

I actually changed to the new selector because Atom 1.17.0 broke the old ones (according to #9). So if you are running anything before 1.17.0 it is possible that this package will not work for you and you'll need to use version 1.0.10 of this package, as long as you are using an older atom version

EDIT: You can downgrade vo v1.0.10 using `apm install click-link@1.0.10`

**Comment by arvinep** [comment]
Alright upgrading Atom to 1.18.0 fixed the issue with click-link 1.0.11.
I was running 1.16.0 and click-link 1.0.10 was working fine with it.

Thanks and cheers,
